packages/libsigcplusplus.rb

```diff
@@ -3,33 +3,46 @@
 class Libsigcplusplus < Package
   description 'libsigc++ implements a typesafe callback system for standard C++.'
   homepage 'http://libsigc.sourceforge.net/'
-  version '2.10.0'
-  source_url 'http://ftp.gnome.org/pub/GNOME/sources/libsigc++/2.10/libsigc++-2.10.0.tar.xz'
-  source_sha256 'f843d6346260bfcb4426259e314512b99e296e8ca241d771d21ac64f28298d81'
+  version '2.99.10'
+  source_url 'https://github.com/GNOME/libsigcplusplus/archive/2.99.10.tar.gz'
+  source_sha256 '502735363777cd8d540299035daa3f9162912f325491cbb0f8d709b691593e07'
 
   binary_url ({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/libsigcplusplus-2.10.0-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/libsigcplusplus-2.10.0-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/libsigcplusplus-2.10.0-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/libsigcplusplus-2.10.0-chromeos-x86_64.tar.xz',
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/libsigcplusplus-2.99.10-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/libsigcplusplus-2.99.10-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/libsigcplusplus-2.99.10-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/libsigcplusplus-2.99.10-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
-    aarch64: '526748e3a15fcd316fefc8c0359880b5c89640ad89a01de8c3f6210251e229f4',
-     armv7l: '526748e3a15fcd316fefc8c0359880b5c89640ad89a01de8c3f6210251e229f4',
-       i686: 'cbdf640dbf2462b3b8f24bb5f7cd2345f1ff3f46c983191e7ee31482213d24cd',
-     x86_64: 'c32ff29b0f4a8394b69502cd3dc030848f84c5c3df048beb88966f7b1b6c6935',
+    aarch64: 'a8b8bdd9040ebb67bb092055ee36a25135825d08a039f4596e835996bb245cff',
+     armv7l: 'a8b8bdd9040ebb67bb092055ee36a25135825d08a039f4596e835996bb245cff',
+       i686: '57e89d0057217ae81a55ec675be843ed5dbfa54d5ad1deba4b900c6b9c72c451',
+     x86_64: 'da7e1a5edf563ae95c677b89283a2f79cc23380a1b58b264df1abfcfec919bca',
   })
 
   depends_on 'pkgconfig' => :build
   depends_on 'diffutils' => :build
   depends_on 'm4' => :build
+  depends_on 'gcc7' => :build  # c++14 support
+  depends_on 'cmake' => :build
 
   def self.build
-    system "./configure --prefix=#{CREW_PREFIX} --libdir=#{CREW_LIB_PREFIX}"
-    system "make"
+    system "mkdir -p build"
```

Comment:
No need for the `-p` flag since it's a single directly but it doesn't hurt including it.

Reply:
@uberhacker gcc7.3 should work I think